templates/agenda/index.html.twig

```diff
@@ -9,9 +9,11 @@
         <div class="col">
             <h1>{% trans %}Agendas{% endtrans %}</h1>
         </div>
-        <div class="col-auto">
-            <a name="" id="" class="btn btn-success" href="create" role="button">{% trans %}Create agenda{% endtrans %}</a>
-        </div>
+        {% if not is_granted('ROLE_BOARD_MEMBER') %}
+            <div class="col-auto">
+                <a name="" id="" class="btn btn-success" href="create" role="button">{% trans %}Create agenda{% endtrans %}</a>
+            </div>
+        {% endif %}
```

Comment:
Use `{% if is_granted('ROLE_CASEWORKER') %}` to check that the user is allowed to create agendas rather than not not allowed (sic!).

Are controller actions updated to check that the user is allowed to do what the controller action does?

Remove the empty `name` and `id` attributes.

Reply:
Removed empty `name` and `id` attributes in 6041fd4b3c98b401a5f20b25d988e150beefd841.

Reply:
Checked that user is allowed rather than not not allowed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.

Reply:
Added permission checks in controllers in e9da2ed66dd1211ed8012af4e526b259e2ddeefe.